packager/media/base/playready_pssh_generator.cc

```diff
@@ -59,6 +57,45 @@ std::vector<uint8_t> ConvertGuidEndianness(const std::vector<uint8_t>& input) {
   return output;
 }
 
+void ReplaceString(std::string* str,
+                   const std::string& from,
+                   const std::string& to) {
+  size_t location = str->find(from);
+  if (location != std::string::npos) {
+    str->replace(location, from.size(), to);
+  }
+}
+
+void AesEcbEncrypt(const std::vector<uint8_t>& key,
+                   const std::vector<uint8_t>& plaintext,
+                   std::vector<uint8_t>* ciphertext) {
+  CHECK_EQ(plaintext.size() % AES_BLOCK_SIZE, 0u);
+  ciphertext->resize(plaintext.size());
```

Comment:
So for ECB, the output size does not need to be one block more than the input size?

Reply:
I double-checked the tests with valgrind, and got no errors.  Then I noticed that this code is not actually covered by tests...

I'll add an extra block.  :-(

I think we will need to do some code coverage analysis at some point.  I'll file an issue for that, too.